test/controllers/GuidanceControllerSpec.scala

```diff
@@ -1133,7 +1133,7 @@ class GuidanceControllerSpec extends BaseSpec with ViewFns with GuiceOneAppPerSu
 
```

Comment:
Strictly speaking the update is in  GuidanceService so there should be a new or updated test in GuidanceServiceSpec.